src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -81,8 +80,9 @@ const RadioFrequencyLink = styled(Link)`
   ${({ service }) => service && getSansRegular(service)};
 `;
 
-const CanonicalRadioSchedule = ({ endpoint }) => {
-  const [schedule, setRadioSchedule] = useState();
+const CanonicalRadioSchedule = ({ endpoint, initialData }) => {
+  const radioScheduleData = useData(endpoint, initialData);
```

Comment:
Would we maybe want a loading boolean and error in a similar fashion to `@apollo/react-hooks` ?
```js
const { loading, error, data } = useQuery(EXCHANGE_RATES);
```
We might have a need for loading animations at some point, maybe when personalisation comes in.

Reply:
Yeah I think that would be a good idea!